src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+export const TodoModal: React.FC<{
+  todo: Todo;
```

Comment:
Here it is readable, but having 3+ props may become a bit messy. 

Better approach for such cases will be to simply define a Props type, like:

```
type Props = {
    todo: Todo,
    ...
}
```

and use Props at the component definition.